gorp_test.go

```diff
@@ -2360,6 +2361,31 @@ func TestPrepare(t *testing.T) {
 	}
 }
 
+func TestWithCanceledContext(t *testing.T) {
+	if _, driver := dialectAndDriver(); driver != "mysql" {
+		t.Skip("Cancellation is not yet supported by all drivers. Known to be supported by mysql")
+	}
+
+	dbmap := initDbMap()
+	defer dropAndClose(dbmap)
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	defer time.AfterFunc(100 * time.Millisecond, cancel).Stop()
```

Comment:
I suppose you did that on purpose, just wanted to point out that this is equivalent to:
```go
ctx, cancel := context.WithTimeout(context.Background(), 100*time.Millisecond)
defer cancel()
```